ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -360,7 +360,7 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-localzone-byo-subnet-ovn-day2
-- as: aws-ipi-localzone-byo-subnet-sdn-f14
+- as: aws-ipi-localzone-byo-subnet-ovn-f14
```

Comment:
we are going to drop snd coverage in localzone testing, right?

Reply:
Yes, OpenShiftSDN support will be removed in 4.15, see https://github.com/openshift/installer/pull/7765#discussion_r1410398696

Reply:
got it.